Earthfile

```diff
@@ -0,0 +1,21 @@
+VERSION 0.8
+
+nanoc:
+	#FROM docker.io/library/ruby:3.3-alpine
+	FROM docker.io/library/ruby:3.3
+	COPY --dir content layouts lib Gemfile Gemfile.lock Rules nanoc.yaml \
+		/src/site/
+	WORKDIR /src/site
+
+
+build:
+	FROM +nanoc
+	RUN bundle install
+	RUN bundle exec nanoc
+	SAVE ARTIFACT output AS LOCAL output
+
+
+bundle-update:
+	FROM +nanoc
+	RUN bundle update
+	SAVE ARTIFACT Gemfile.lock AS LOCAL Gemfile.lock
```

Comment:
What's the use case of this stage? I don't see it referenced on `ci.yaml`

Reply:
This can be used to run bundle-update inside the build container and save the result back to disk. When I was working on Alpine support I played with it. I'm alright dropping it if you prefer.

Reply:
Do you think it might be useful for local purposes? In that case, we can keep it

Reply:
I think it's worth keeping in order to allow developers the option of running update without having a local Ruby toolchain ready.